eras/babbage/formal-spec/transactions.tex

```diff
@@ -5,7 +5,8 @@ \section{Transactions}
   %
   \begin{equation*}
     \begin{array}{l@{\qquad=\qquad}lr}
-      \TxOut & \Addr \times \Value \times (\hldiff{\Datum} \uniondistinct \DataHash)^? \times \hldiff{\Script^?}
+      \TxOut & \Addr \times \Value \times (\hldiff{\Datum} \uniondistinct \DataHash)^? \times \hldiff{\Script^?} \\
+      \ScriptPurpose & \PolicyID \uniondistinct \TxIn \uniondistinct \AddrRWD \uniondistinct \DCert
```

Comment:
@TimSheard was adamant that I remind the reader here what a `ScriptPurpose` was, even though it has not changed :)